doc/authoring_help.md

```diff
@@ -13,8 +13,10 @@ To override help for a given command:
 	2. Search result: src/command_modules/azure-cli-**profile**/azure/cli/command_modules/**profile**/commands.py.
 	3. Result shows "account clear" is in the "profile" module.
 2. Using the module name, find the YAML help file which follows the path pattern:
-	1.  src/command_modules/azure-cli-**[module name]**/azure/cli/command_modules/**[module name]**/_help.py.
-	2.  If the file doesn't exist, it can be created.
+	1.  src/command_modules/azure-cli-**[module name]**/azure/cli/command_modules/**[module name]**/_help.py
+	2.  src/command_modules/azure-cli-**[module name]**/azure/cli/command_modules/**[module name]**/help.yaml
+	    - **Pure Yaml format. Coming Soon**
```

Comment:
Remove this. It will be supported as soon as this is merged, so it isn't coming "soon"

Reply:
Sounds good, I just wanted to hint at this so people won't use help.yaml until we have resolved the extension issue.